helpers/class.TestSession.php

```diff
@@ -101,6 +101,11 @@ class taoQtiTest_helpers_TestSession extends AssessmentTestSession
      */
     private $lock = null;
 
+    /**
+     * @var boolean
```

Comment:
can you provide description as you did in the setReadOnly method, or move it here?